palantirnet/st-bernards
# MSSB-36: update config from prod

## JIRA Ticket(s)

- See ticket [MSSB-36: Assess needed contrib module updates](https://palantir.atlassian.net/browse/MSSB-36).

## Description

Updates configuration from prod in preparation for module updates.

## Testing instructions

- Import the production database from 1/17/20.
- `drush cr`
- `drush updb`
- `drush cr`
- `drush cim`
- The only changes you should see are for the configuration splits.
- Verify the site works as expected.
